src/Security/Authentication/Core/src/Events/RemoteAuthenticationContext.cs

```diff
@@ -42,8 +42,16 @@ public abstract class RemoteAuthenticationContext<TOptions> : HandleRequestConte
         /// </summary>
         public void Success() => Result = HandleRequestResult.Success(new AuthenticationTicket(Principal!, Properties, Scheme.Name));
 
+        /// <summary>
+        /// Calls <see cref="HandleRequestResult.Fail(Exception)"/> indicating that authentication failed.
```

Comment:
That's an implementation detail that's not relevant to the caller.
```suggestion
        /// Indicates that authentication failed.
```